lib/Doctrine/DBAL/Types/BlobType.php

```diff
@@ -46,7 +46,7 @@ public function convertToPHPValue($value, AbstractPlatform $platform)
         }
 
         if (is_string($value)) {
-            $fp = fopen('php://temp', 'rb+');
+            $fp = tmpfile();
```

Comment:
Correct. But allow_url_fopen = Off causes the fopen not work. Perhaps an alternative is checked before the state allow_url_fopen.